### PR TITLE
04 02 2024  as clear okta network zone list

### PR DESCRIPTION
**Pull request by AcceleryntSecurityDev**
Change(s):
   - New Playbook

   Reason for Change(s):
   - New Playbook

   Version Updated:
   - N/A

   Testing Completed:
   - Yes
image

### CODE REVIEW

File: Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,394 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
```
Comment (v-prasadboke) [inline Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json:4]: Hello @AcceleryntSecurityDev, 
Postdeploymentsteps are missing from metadata. Can you please add that

Comment (AcceleryntSecurityDev) [reply Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json:4]: @v-prasadboke this has been added

File: Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+        "title": "AS-Clear-Okta-Network-Zone-List", 
+        "description": "This playbook is intended to be run on a schedule. It will clear out all but one of the IPs from an Okta Network Zone list (leaving at least one entry is required by API). This will prevent the list from reaching capacity and preventing newer entries from being added.",
+        "prerequisites": "1. An Okta admin account is needed, along with an API token. 2. A Microsoft Azure key vault containing the Okta API token must also be set up. Support for the set up and configuration of each of these items can be found here: https://github.com/Accelerynt-Security/AS-Clear-Okta-Network-Zone-List",
+        "postDeployment": "1. Access to the Azure Key Vault secret must be granted to the playbook."
```
Comment (v-prasadboke) [inline Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json:8]: Please add , at the end. 
String not closed properly. ' , '  is expected at the end

Comment (AcceleryntSecurityDev) [reply Playbooks/AS-Clear-Okta-Network-Zone-List/azuredeploy.json]: @v-prasadboke this is fixed

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**